### ISSUE TITLE
patterplate build --base not working

### ISSUE THREAD

**Issue by pixelass** [patternplate/patternplate#177]
the following commands do not work

```shell
patternplate build --base=/patterns
patternplate build --base=patterns/
patternplate build --base=/patterns/
patternplate build --base=patterns
patternplate build --base=my-repo/patterns
```

## Expected result

URLs use the base as a prefix `my-repo/patterns?query`

## Actual result

The URLs are always absolute `?query`

## Versions

```
    "@patternplate/cli": "^2.0.0-55",

```

**Comment by pixelass** [comment]
https://github.com/patternplate/patternplate/blob/master/packages/cli/src/build/index.js#L25-L27

```js
const base = flags.base === "/"
    ? flags.base
    : `/${typeof flags.base === "undefined" ? path.basename(out) : "/"}`;
```

if base is `/` allow it
otherwise `/${path.basename(out)}` or `//` 

Not sure if this makes sense.

**Comment by pixelass** [comment]
should probably be 

```js
    base = flags.base === "/" ? flags.base : `/${typeof flags.base === "undefined" ? path.basename(out) : flags.base}`;

```

tested locally

**Comment by marionebl** [comment]
Went with an easier decision tree that requires you to specify `--base` always. Fixed via `2.0.0-57`
